client/src/rpb/index.js

```diff
@@ -276,20 +277,17 @@ class RPB extends React.Component {
                 <div
                   tabIndex={-1}
                   id="modal-child"
-                  className="container app-font"
-                  style={{
-                    backgroundColor: "white",
-                    overflow: "auto",
-                    lineHeight: 1.5,
-                    padding: "0px 20px 0px 20px",
-                    borderRadius: "5px",
-                    fontWeight: 400,
-                  }}
+                  className="container app-font modal-container"
                 >
-                  <TablePicker
-                    onSelect={(id) => this.pickTable(id)}
-                    broken_url={broken_url}
-                  />
+                  {!dataset_type && <Step1 />}
+                  {/*dataset_type && !selected_subject && <Step2/>*/}
+                  {dataset_type /* && selected_subject */ && (
```

Comment:
Here and a bit above there's some commented out stuff. Looks like this `Step2` stuff's currently not used? While I'm at it, `Step1` and `Step2` are kind of vague names.

Reply:
Ah yes, so I was considering making it a 2 step table picker process, with selecting organizations but saw some disadvantages to it. I'll drop this completely